libmproxy/script/script.py

```
@@ -88,6 +88,8 @@ def run(self, name, *args, **kwargs):
             Raises:
                 ScriptException if there was an exception.
         """
+        if not self.ns:
```

This may hide errors at other places - do we want to raise a ScriptError here?